src/pages/host/Metadata.tsx

```diff
@@ -26,7 +26,7 @@ export const Metadata: React.FC<{
   const runningTaskId = runningTask?.id;
   const runningTaskName = runningTask?.name;
 
-  const taskLink = `${getUiUrl()}/task/${runningTaskId}`;
+  const taskLink = `/task/${runningTaskId}`;
```

Comment:
We have a utility function for this
https://github.com/evergreen-ci/spruce/blob/82684a550fe376bf2661be5fb74c4cd911941707/src/constants/routes.ts#L94